library/src/main/java/me/grantland/widget/AutofitHelper.java

```diff
@@ -433,12 +432,31 @@ public AutofitHelper setEnabled(boolean enabled) {
 
             if (enabled) {
                 mTextView.addTextChangedListener(mTextWatcher);
-                mTextView.addOnLayoutChangeListener(mOnLayoutChangeListener);
+
+                if ( android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
+                    if (mOnLayoutChangeListener == null){
+                        mOnLayoutChangeListener = new View.OnLayoutChangeListener() {
+                            @Override
+                            public void onLayoutChange(View v, int left, int top, int right, int bottom, int oldLeft, int oldTop, int oldRight, int oldBottom) {
+                                autofit();
+                            }
+                        };
+                    }
+                    mTextView.addOnLayoutChangeListener(mOnLayoutChangeListener);
+                } else {
+                    if ( mTextView instanceof AutofitTextView)
+                        ((AutofitTextView) mTextView).setSizeListener(sizeListener);
```

Comment:
Probably, I've done this a while back so it's hard to rember exactly, I only needed it for AutofitTextView